### PR TITLE
[refactor] modify_some_state_to_interface

### PR DESCRIPTION
**Pull request by Zzm0809**
## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dinky-web/src/pages/RegCenter/Alert/AlertGroup/index.tsx [hunk 1]
```diff
@@ -45,21 +47,18 @@ const AlertGroupTableList: React.FC = (props: any) => {
...
   /**
    * execute query alert instance list
    * set alert instance list
    */
   const queryAlertGroupList = async () => {
-    await queryList(API_CONSTANTS.ALERT_GROUP).then((res) => {
-      setAlertGroupList(res.data);
-    });
+    await queryList(API_CONSTANTS.ALERT_GROUP).then((res) =>
+      setAlertGroupState((prevState) => ({ ...prevState, alertGroupList: res.data }))
```
Comment (leechor) [inline dinky-web/src/pages/RegCenter/Alert/AlertGroup/index.tsx:60]: For this, only using one of `await` or 'Promise` is better, recommend using `await`

**Review by leechor — APPROVED** [review]
LGTM

**Review by aiwenmo — APPROVED** [review]
LGTM

**Review by leechor — APPROVED** [review]
LGTM

**Pull request merged**